plugins/action/configure.py

```diff
@@ -118,6 +118,9 @@ def run(self, tmp=None, task_vars=None):
         result = super(ActionModule, self).run(tmp, task_vars)
 
         json_config = json.loads(self._task.args.get("config") or {})
+        if isinstance(json_config, dict):
```

Comment:
I think we should change the data type of config option to `dict` (similar to the resource modules)